src/__test__/DateTimePicker.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import { DateTimePicker } from "../components";
+import userEvent from "@testing-library/user-event";
+
+
+test("correctly renders styles", () => {
+  const hello = jest.fn();
+  const { getByTestId } = render(<DateTimePicker onChange={hello} />);
+  const DateRangeElement = getByTestId("date-time-picker");
+  expect(DateRangeElement).toHaveStyle({ width: "15rem;" });
```

Comment:
Does this test actually pass? Because this check is testing for the inline style, but you are applying it with a class.

Reply:
Double checked and the test does pass